### PR TITLE
Added in UFO QC check

### PR DESCRIPTION
**Pull request by lliu1220**
Added in UFO QC check by fixing:

 if (val[jobs] == missing || ref[jobs] == missing || ref[jobs] == 0.0) {

to allow ref 0.0 to be opted out

changes are:
QCmanager.cc
RatioCheck.cc
CMakelist.txt
instantiateObsFilterFactory.h

image
image

### CODE REVIEW

File: src/ufo/filters/RatioCheck.cc_ORG [hunk 1]
```diff
@@ -0,0 +1,95 @@
+/*
```
Comment (aerorahul) [inline src/ufo/filters/RatioCheck.cc_ORG:1]: Where is this compiled/used?

**Review by aerorahul — CHANGES_REQUESTED** [review]

File: src/ufo/filters/RatioCheck.cc_ORG (hunk 1 above)
Comment (lliu1220) [reply src/ufo/filters/RatioCheck.cc_ORG]: removed...

**Review by emilyhcliu — APPROVED** [review]

**Review by emilyhcliu — CHANGES_REQUESTED** [review]
Remove the extra file that does not belong to the repository.
Changes related to HW look good!

**Comment by lliu1220** [comment]
> Remove the extra file that does not belong to the repository.
> Changes related to HW look good!

Removed with commit 362669e6d4911563126c741964a1fb15dc0228d6, thank you!

**Review by emilyhcliu — APPROVED** [review]

**Pull request closed without merging**